backend/sites/src/app/resolvers/cart/cart.resolver.ts

```diff
@@ -51,7 +51,7 @@ export class CartResolver {
         this.sitesLogger.log('CartResolver.getCartItemsForUser() RES:422 end');
```

Comment:
please correct the response code in the logs

Reply:
Done